modules/index.js

```diff
@@ -30,8 +31,13 @@ module.exports.messageRoutes = {
   ">betta": flickr.betta,
   ">image": flickr.imageSearch,
 
+  // danbooru
+  ">safe": danbooru.safe,
+  ">ecchi": danbooru.ecchi,
+  ">nsfw": danbooru.nsfw,
+
   // Random
   ">dice": random.dice,
   ">roll": random.dice,
-  ">pick": random.pick
+  ">pick": random.pick,
```

Comment:
In JavaScript you're not supposed to comma the last variable in an object. This is only corrected in newer versions. Please revert.